### DISCUSSION TITLE
What is the name of an 🇮🇳 dish made of a very thin, crepe-like batter that can be stuffed?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#733] [Galaxy Trivia] [answered]
**Bonus:** List main ingredients.

**Comment by peterbaccaro** [comment]
The Indian dish made of a very thin, crepe-like batter that can be stuffed is a **dosa**.
A **dosa** is a thin pancake in South Indian cuisine made from a fermented batter of ground black lentils and rice. 
**Dosas** are popular in South Asia as well as around the world.

**Comment by thuongtruong1009** [comment]
The dish you are referring to is called "Dosa." Dosa is a popular South Indian dish made from a fermented batter consisting of rice and black gram (lentils). The batter is spread thinly on a hot griddle or pan to form a crepe-like texture. It is typically served with various fillings and chutneys, such as potato masala, coconut chutney, and sambar. Dosa is known for its crispy texture and is enjoyed as a breakfast or snack item in many parts of India and beyond.

**Comment by SANKARAMDAS** [comment]
"Dosa" an Indian (South Indian Crepes), a crepe or thin pancake made from a fermented batter consisting of lentils and rice. It originated in South India where it has been enjoyed for many centuries.
Ingredients used to prepare dosa are rice, urad dal, salt and water
Dosa is high in carbohydrates and contains no added sugars or saturated fats. As its key ingredients are rice and black gram, it is a good source of protein. A typical homemade plain dosa without oil contains about 112 calories, of which 84% is carbohydrate and 16% protein.
The dosa batter ingredients are rice and urad dal in the ratio 4:1.

**Comment by techvmuppa** [comment, marked as answer]
The name of the Indian dish made of a very thin, crepe-like batter that can be stuffed is called "dosa". Dosa is a popular South Indian dish that is made from a fermented batter of rice and black gram (a type of lentil). The batter is spread thinly on a hot griddle or tawa and cooked until crispy and golden brown. Dosas can be stuffed with a variety of fillings such as spiced potatoes, onions, or cheese, and are often served with coconut chutney and sambar (a type of lentil soup). Dosas are a staple food in South India and have gained popularity across the country and beyond due to their delicious taste and versatility.
